cmd/auth/login.go

```diff
@@ -1,19 +1,26 @@
 package auth
 
 import (
+	"github.com/pingidentity/pingctl/cmd/common"
 	"github.com/spf13/cobra"
 )
 
 func NewLoginCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "login",
-		Short: "Login user to the CLI",
-		Long:  "Login user to the CLI",
-		RunE: func(cmd *cobra.Command, args []string) error {
-			// authConnectors := []connector.Authenticatable{}
-			return nil
-		},
+		Args:                  common.ExactArgs(0),
+		DisableFlagsInUseLine: true, // We write our own flags in @Use attribute
+		Long:                  "Login user to the CLI",
+		RunE:                  authLoginRunE,
+		Short:                 "Login user to the CLI",
+		Use:                   "login [flags]",
 	}
 
 	return cmd
 }
+
+func authLoginRunE(cmd *cobra.Command, args []string) error {
```

Comment:
What are these RunE methods? I guess these just aren't implemented yet? If these are visible to the user it might be good to have some message about these being not implemented.